### PR TITLE
ES2015ify and support Node.js >=4

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,19 +28,14 @@
     "osx"
   ],
   "dependencies": {
+    "execa": "^0.6.3",
     "lowercase-first-keys": "^1.0.0",
     "obj-type": "^1.0.0",
-    "pify": "^2.2.0",
-    "pinkie-promise": "^1.0.0",
-    "simple-plist": "0.0.4"
+    "pify": "^3.0.0",
```
Comment (gillstrom) [inline package.json]: This is not needed anymore right?

Comment (kevva) [reply package.json]: Nope.

**Comment by gillstrom** [comment]
Nice job bre

**Comment by kevva** [comment]
Good merge boy.

**Pull request merged**